### ISSUE TITLE
How to calculate IOU for the model with pretrained cityScape weights

### ISSUE THREAD

**Issue by nullbringer** [bonlime/keras-deeplab-v3-plus#126]
Hi,
Can anyone give any pointers on how to calculate IOU for this model?
I am using cityScape weights.

The labels do not match!
